templates/vhost/_wsgi.epp

```diff
@@ -0,0 +1,50 @@
+<% if $wsgi_application_group or $wsgi_daemon_process or $wsgi_import_script or $wsgi_process_group {-%>
+
+  ## WSGI configuration
+<% } -%>
+<% if $wsgi_application_group {-%>
+  WSGIApplicationGroup <%= $wsgi_application_group %>
+<% } -%>
+<% if type($wsgi_daemon_process) =~ Type[String] and $wsgi_daemon_process_options {-%>
```

Comment:
AFAIK you don't need to use `type()` and `Type`.
```suggestion
<% if $wsgi_daemon_process =~ String and $wsgi_daemon_process_options {-%>
```

Reply:
done